gcalbot/gcalbot/account.go

```diff
@@ -180,10 +180,11 @@ func (h *Handler) deleteAccount(accountID string) error {
 		switch err := err.(type) {
 		case nil:
 		case *googleapi.Error:
-			if err.Code == 404 {
-				// channel not found, so don't worry about stopping it
-				continue
+			// if the channel didn't exist, don't error
+			if err.Code != 404 {
+				return err
```

Comment:
having an extra `return nil` here might is probably most clear

Reply:
In which case? If the error is nil, it should continue looping rather than returning. Maybe an explicit `continue` would be good?

Reply:
ah yes explicit continue would be good then, but comment applied to a few places in this diff

Reply:
I'll take a look, thanks

Reply:
What do you think about my changes?

Reply:
can't this just be 
```
if err.Code == 404 {
    continue
}
return err
```